src/modules/roc_fec/writer.cpp

```diff
@@ -147,6 +146,23 @@ void Writer::write(const packet::PacketPtr& pp) {
     }
 }
 
+bool Writer::resize_repair_block_(size_t rblen) {
+    if (repair_packets_.size() == rblen) {
+        return true;
+    }
+
+    if (repair_packets_.resize(rblen)) {
```

Comment:
Lets rename `repair_packets` into `repair_block`? Because field name and function name seems to be inconsistent a little bit.

Reply:
Well, we can leave it as is for now because we have some functions that have `block` in their name and other functions that have `packets` in their name, but packets and block are the same concept in the current context.

Reply:
Renamed for consistency with reader.